starport/pkg/cosmosanalysis/module/module_test.go

```diff
@@ -4,9 +4,67 @@ import (
 	"context"
 	"testing"
 
-	"github.com/kr/pretty"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/tendermint/starport/starport/pkg/protoanalysis"
 )
 
-func TestExampleDiscover(t *testing.T) {
-	pretty.Println(Discover(context.Background(), "/home/ilker/Documents/code/src/github.com/tendermint/starport/local_test/mars"))
+func TestDiscover(t *testing.T) {
+	type args struct {
+		sourcePath string
+		protoDir   string
+	}
+	tests := []struct {
+		name string
+		args args
+		want []Module
+	}{
+		{
```

Comment:
can we add a test case here to actually compare a valid `[]Module` value found after module discovery?